05-html-css/flexbox-intro.md

```diff
@@ -0,0 +1,109 @@
+# Intro to Alignment and Positioning with Flexbox
+
+## Learning Goals
+- Understand the historical context of flexbox compared to other methods of aligning and positioning with CSS
+- Review how the browser uses _flow_ to place elements on the page
+- Change how elements flow on the page using CSS's `display: flex` property
+- Change how elements align on the page using CSS's `display: flex` property
+- Know browser compatibility for flexbox
+
+## History
+
+Historically, _aligning_ items while working with page flow using CSS has been difficult. To align an item within along one dimension (vertically or horizontally), CSS had a lot of attributes and values that weren't great solutions.
+
+By default, every HTML tag comes with a `display` property value of `inline` or `block`, which determine how they flow on the page but was typically only left to right, top to bottom. Also, by default every element aligns the baseline of itself with the baseline of the parent element. Vertical alignment in CSS is notoriously complex.
+
+In comes _flexbox_ (stylistically lowercase, sometimes "flex-box", short for Flexible Box Module).
+
+Flexbox has the following advantages:
+1. Easier, more intuitive attributes
```

Comment:
Can we reframe to exclude "easier" but provide the context in which this is more intuitive?